src/index.ts

```diff
@@ -186,50 +180,40 @@ class CruxPayPeer {
     }
 
     public async init() {
+        await this._setupConfigService();
+        if (!this._configService) {
```

Comment:
In what scenario will this be the case?

Reply:
this statement is added to tell the linter that _configService is assumed to be present.